spec/delivery_spec.rb

```diff
@@ -1,9 +1,11 @@
 require 'rspec'
 require 'delivery'
 
-describe Delivery do
-  it 'states order delivery time' do
-    time = (Time.now + 1*60*60).strftime("%H:%M")
-    expect(subject.delivery_time).to eq "You will recieve your order at or before #{time}."
+RSpec.describe Delivery do
+  describe '#delivery_time' do
+    it 'states order delivery time' do
+      time = (Time.now + 1 * 60 * 60).strftime("%H:%M")
```

Comment:
This is awesome! I wouldn't have known how to implement the time to be an hour from now, its cool to see!